public/angular/js/controllers/media/settings/settings_modal.js

```diff
@@ -0,0 +1,49 @@
+angular.module('Aggie')
+
+.controller('MediaSettingsModalInstanceController', [
+  '$scope',
+  '$modalInstance',
+  'Settings',
+  'FlashService',
+  'settingsValues',
+  'media',
+  '$filter',
+  function($scope, $modalInstance, Settings, flash, settings, media, $filter) {
+
+    $scope.settings = settings;
+    $scope.media = media;
+
+    $scope._showErrors = false;
```

Comment:
We have different "logics". The blank separates the settings related variables from the other one. It made sense when I was developing, but maybe it doesn't in the big picture of the novel